samples/drivers/gpio/src/main.c

```diff
@@ -163,9 +184,17 @@ void main(void)
 	}
 
 	/* Setup GPIO input, and triggers on rising edge. */
+#ifdef CONFIG_SOC_CC2650
+	ret = gpio_pin_configure(gpio_dev, GPIO_INT_PIN,
+				 (GPIO_DIR_IN | GPIO_INT |
+				  GPIO_INT_EDGE | GPIO_INT_ACTIVE_HIGH |
+				  GPIO_INT_DEBOUNCE | GPIO_INT_PIN_EXTRAFLAGS));
```

Comment:
Why not just replace GPIO_INT_PIN_EXTRAFLAGS with GPIO_PUD_PULL_UP, not sure what the #define is getting us.